src/classes/AccountDeDuper.cls

```diff
@@ -4,11 +4,17 @@ public with sharing class Account_DeDuper {
 Public AccountDe_Duper () {
 
 }
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
+public void BADSTUFF__4() {}
+
 Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName, String pPostCode, String pCity, id pRecordTypeId, Boolean pShowAllRecordTypes, Integer pSearchLimit) {
```

Comment:
This method has a very long parameter list. Consider encapsulating all related parameters into a single object, or consider passing some parameters in the constructor prior to the invocation of this method.